it/helpers/servicemocks/IncomeTaxViewChangeStub.scala

```diff
@@ -184,4 +185,14 @@ object IncomeTaxViewChangeStub { // scalastyle:off number.of.methods
   def stubGetPaymentAllocationResponse(nino: String, paymentLot: String, paymentLotItem: String)(status: Int, response: JsValue): Unit =
     WiremockHelper.stubGet(paymentAllocationChargesUrl(nino, paymentLot, paymentLotItem), status, response.toString())
 
+  // Repayment History By RepaymentId stubs
```

Comment:
Please update to remove the commented line

Reply:
It's just the comment for the Stubs section.